egs2/voxtlm/lm1/local/prepare_bpe_text.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-i", "--inp", type=str, help="Input file with all training data"
```

Comment:
```suggestion
        "-i", "--input", type=str, help="Input file with all training data"
```
We usually use more complete words or phrases for `--` based options